asllib/carpenter/dune

```diff
@@ -4,6 +4,8 @@
   (:standard -w -40-42-44-48))
  (modules :standard \ main)
  (optional)
+ (enabled_if
+  (>= %{ocaml_version} 4.14))
```

Comment:
I am not very keen on it, but does this mean that carpenter code is not compatible with OCaml < 4.14?

Reply:
No big deal.